content/en/tutorials/elb-load-balancing/index.md

```diff
@@ -0,0 +1,325 @@
+
+Let us now move to the `serverless.yml` file and specify our deployment bucket and the functions we want to deploy. Our initial configuration in the `serverless.yml` file should look like this:
+
+```yaml
+service: serverless-elb
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  deploymentBucket:
+    name: testbucket
+
+functions:
+  hello1:
+    handler: handler.hello1
+    events:
+    - alb:
+        listenerArn: !Ref HTTPListener
+        priority: 1
+        conditions:
+          path: /hello1
+  hello2:
+    handler: handler.hello2
+    events:
+    - alb:
+        listenerArn: !Ref HTTPListener
+        priority: 2
+        conditions:
+          path: /hello2
+```
+
+In the above configuration, we specify our Lambda functions `hello1` and `hello2`, and specify an HTTP listener that forwards requests to the target group. We have also specified a `deploymentBucket` property, which stores the deployment artifacts. Before deploying our Serverless project, we would need to create an S3 bucket named `testbucket` in LocalStack. 
+
+Let us now create a VPC, a subnet, an Application Load Balancer, and an HTTP listener on the load balancer, which redirects the traffic to the target group. We can do this by adding the following resources to our `serverless.yml` file:
+
+```yaml
+resources:
+  Resources:
+    LoadBalancer:
+      Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+      Properties:
+        Name: lb-test-1
+        Subnets:
+          - !Ref Subnet
+    HTTPListener:
+      Type: AWS::ElasticLoadBalancingV2::Listener
+      Properties:
+        DefaultActions:
+          - Type: redirect
+            RedirectConfig:
+              Protocol: HTTPS
+              Port: 443
+              Host: "#{host}"
+        LoadBalancerArn: !Ref LoadBalancer
+        Protocol: HTTP
+    Subnet:
+      Type: AWS::EC2::Subnet
+      Properties:
+        VpcId: !Ref VPC
+        CidrBlock: 12.2.1.0/24
+        AvailabilityZone: !Select
+          - 0
+          - Fn::GetAZs: !Ref "AWS::Region"
+    VPC:
+      Type: AWS::EC2::VPC
+      Properties:
+        EnableDnsSupport: "true"
+        EnableDnsHostnames: "true"
+        CidrBlock: 12.2.1.0/24
+```
+
+With this, we have now completed our Serverless project's configuration. We can now create our local AWS infrastructure on LocalStack and deploy our Application Load Balancers with two Lambda functions as targets.
+
+## Creating the infrastructure on LocalStack
+
+Now that the initial setup is done, we can give LocalStack's AWS emulation a run on our local machine. Let's start LocalStack:
+
+{{< command >}}
+$ LOCALSTACK_API_KEY=<your-api-key> localstack start -d
+{{< / command >}}
+
+First, we create an S3 bucket named `testbucket` in LocalStack to store our deployment artifacts. We can do this by running the following command:
+
+{{< command >}}
+$ awslocal s3api create-bucket --bucket testbucket
```

Comment:
if we create the bucket manually, then I think we can get rid of the `serverless-deployment-bucket` plugin introduced above.. 👍